src/main/kotlin/com/atlassian/performance/tools/ssh/api/Ssh.kt

```diff
@@ -76,7 +77,7 @@ data class Ssh(
     }
 
     private fun prepareClient(): SSHClient {
-        val ssh = SSHClient()
+        val ssh = SSHClient(PerformanceDefaultConfig())
```

Comment:
Any idea why https://github.com/atlassian/ssh/pull/6 doesn't reproduce this?

Reply:
Could be anything - different RNG, different RNG configuration, different entropy source, different JVM, hardware RNG. Hard to tell.

Reply:
Roger that